collect_app/src/androidTest/java/org/odk/collect/android/instrumented/forms/FormUtilsTest.java

```diff
@@ -37,8 +37,8 @@ public FormEntryController create(FormDef formDef) {
 
     @Before
     public void setUp() {
-        rule.startAtFirstLaunch()
```

Comment:
We can use:

```
        rule
                .startAtMainMenu()
                .copyForm(BASIC_FORM);
```

like in other tests and then that `copyForm` from CollectTestRule is not needed at all.

Reply:
I don't understand. Could you give me an example of the change you're thinking about here?

Reply:
After your changes, the `setUp` method looks like this:

```
    @Before
    public void setUp() {
        rule.copyForm(BASIC_FORM)
                .startAtFirstLaunch();
    }
```

and the `copyForm` method used there is defined in `CollectTestRule` and used only in this `FormUtilsTest`. So instead of defining that method in `CollectTestRule` you can use the one defined in `MainMenuPage` as we do in all the other cases.

Then the `setUp` method would be like:
```
    @Before
    public void setUp() {
        rule
                .startAtMainMenu()
                .copyForm(BASIC_FORM);
    }
```

Reply:
Ah I see! Good point. I'm realising we can go a step further and just call `StorageUtils` directly here as we're not interacting with the UI at all in this test (it's an "instrumented" unit test).